newIDE/app/src/AssetStore/ShopTiles.js

```diff
@@ -462,3 +463,49 @@ export const PrivateGameTemplateTile = ({
     </GridListTile>
   );
 };
+
+export const ExampleTile = ({
```

Comment:
It feels similar to other tiles in this file, though this file is named `ShopTiles` so it feels wrong at the same time.
Should we rename it?

Reply:
OK then! they can be considered as free game templates yeah